lib/scala/downloader/src/main/java/org/enso/downloader/http/StringProgressBodyHandler.java

```diff
@@ -64,7 +64,8 @@ public void onSubscribe(Flow.Subscription subscription) {
     public void onNext(List<ByteBuffer> items) {
       for (ByteBuffer item : items) {
         var len = item.remaining();
-        progress.reportProgress(len, total == null ? Option.empty() : Some.apply(total));
+        downloaded += len;
+        progress.reportProgress(downloaded, total == null ? Option.empty() : Some.apply(total));
```

Comment:
Simplify to `Option.apply(total)`.

Reply:
Done in c105401ef75381f88962811dba9cfd03f3171281